phpBB/styles/prosilver/theme/icons.css

```diff
@@ -85,3 +85,26 @@
     line-height: .9;
     height: 12px;
 }
+
+
+blockquote cite:before, .uncited:before {
+	-webkit-font-smoothing: antialiased;
+	-mos-osx-font-smoothing: grayscale;
+	font-family: FontAwesome;
+	font-size: 14px;
+	font-weight: normal;
+	font-style: normal;
+	font-variant: normal;
+	line-height: 1;
+	text-rendering: auto;
+	display: inline-block;
+	padding-right: 5px;
+	content: '\f10d'; /* Font Awesome quote-left */
+}
+
+.rtl blockquote cite:before, .rtl .uncited:before {
+	padding-right: 0;
+	padding-left: 5px;
+	content: '\f10e'; /* Font Awesome quote-right */
```

Comment:
Thanks for the quick reply and continued mentoring. It's all part of the education process and much appreciated as I learn the ins and outs. I think I  understand the technical details of your point 1 and definitely agree with the philosophical point of keeping changes as small as possible and compatible with other styles and end-users who might have made changes.